Authservice/src/main/java/com/example/techversantInfotech/Authservice/utils/ImageProcessingUtils.java

```diff
@@ -0,0 +1,82 @@
+package com.example.techversantInfotech.Authservice.utils;
+
+import com.example.techversantInfotech.Authservice.Dto.UserDto;
+import com.example.techversantInfotech.Authservice.Exception.CompletableFutureException;
+import com.example.techversantInfotech.Authservice.Exception.ImageProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.zip.Deflater;
+import java.util.zip.Inflater;
+
+public class ImageProcessingUtils {
+
+    public static UserDto convertObject(String user) {
+        UserDto userDto = new UserDto();
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            userDto = objectMapper.readValue(user, UserDto.class);
+        } catch (
+                IOException err) {
+            throw new ImageProcessingException("NOT_PROCESSED", "Image is not processed successfully");
+        }
+        return userDto;
+    }
+    public static byte[] compressImage(byte[] data) {
+        Deflater deflater = new Deflater();
+        deflater.setLevel(Deflater.BEST_COMPRESSION);
+        deflater.setInput(data);
+        deflater.finish();
```

Comment:
@jenobjob is this something you could move to the model :thinking: So here you just have to pass the parameters to the model builder method...